intuita-webview/src/jobDiffView/DiffViewer/Container.tsx

```diff
@@ -137,13 +144,24 @@ export const Header = ({
 							</VSCodeButton>
 						))}
 					<div
-						className="flex ml-10 justify-between checkbox-container items-center"
+						className={cn(
+							'viewed-button flex ml-10 justify-between checkbox-container items-center',
+							{
+								disabled: changesAccepted,
+							},
+						)}
 						onClick={(e) => {
+							if (changesAccepted) {
```

Comment:
should we stop event propagation if the changes have been accepted?

Reply:
oh, i can remove that check, `.disabled` class already prevents all mouse interactions with that element